scripts.js

```diff
@@ -1,68 +1,82 @@
-var apiurl, myresult;
-
-headerurl = "https://api.flickr.com/services/rest/?method=flickr.photosets.getPhotos&api_key=bc1b5549bb490dcce777b6659b9740f7&photoset_id=72157655762735509&privacy_filter=1&per_page=1&format=json&nojsoncallback=1";
-
-apiurl =
-  "https://api.flickr.com/services/rest/?method=flickr.galleries.getPhotos&api_key=bc1b5549bb490dcce777b6659b9740f7&gallery_id=72157690043876864&per_page=8&format=json&nojsoncallback=1";
-
+var myResult;
+var headerUrl =
+  'https://api.flickr.com/services/rest/?method=flickr.photosets.getPhotos&api_key=bc1b5549bb490dcce777b6659b9740f7&photoset_id=72157655762735509&privacy_filter=1&per_page=1&format=json&nojsoncallback=1';
+var apiUrl =
```

Comment:
`apiUrl` can be declared and assigned in the same line. You're not waiting for a request or function call to get the value.